backend/lib/models/Comment.js

```diff
@@ -1,54 +1,53 @@
-const { model, Schema } = require("mongoose");
+const { Schema, model, ObjectId } = require("mongoose");
+const { schema: authorSchema } = require("./Author");
```

Comment:
Hi @Naraujo13 why are we importing the `Author.schema` if we aren't using it (same as in model/Post.js actually)? Should it be `author: authorSchema` instead of `author: Object` in the comment/post schema definition?

Reply:
Good catch, I've done this at posts also.
The validation for nested documents (author is nested inside comment in this case, and the location is another nested level at author) was giving some problems so I've thought of leaving then as plain objects and adding the authors and locations schema references in a later PR. This way we can have endpoints available to start the integrations with frontend and speedup MVP (while they integrate we can work on these improvements).

Reply:
Sounds good to me. I'll take a more holistic look tomorrow.